src/main/java/Domain/Facades/ShopFacade.java

```diff
@@ -76,7 +80,7 @@ public void closeShop(Integer shopId, String userName) throws Exception {
                 throw new Exception(String.format("Shop ID: %d does not exist.", shopId));
             else {
                 Shop shopToClose = getShopByShopId(shopId);
-                if (shopToClose.checkPermission(userName, Permission.FOUNDER)) {
+                if (shopToClose.checkPermission(userName, Permission.FOUNDER) || _userFacade.isAdmin(userName)) {
```

Comment:
not only the admin of the system can close the shop- fix that pls

Reply:
admin AND shop founder